src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -4,18 +4,34 @@
 import java.awt.event.ActionListener;
 
 import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.factory.FigureFactory;
+import edu.kis.powp.jobs2d.factory.RectangleFactory;
+import edu.kis.powp.jobs2d.factory.TriangleFactory;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
 
 public class SelectTestFigureOptionListener implements ActionListener {
 
 	private DriverManager driverManager;
+	private final static String TERN_T = "Tern test";
+	private final static String RECTANGLE = "Rectangle test";
+	private final static String TRIANGLE = "Triangle test";
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider shared constants.